packages/mux-player/package.json

```diff
@@ -35,29 +35,11 @@
       "require": "./dist/index.cjs.js",
       "default": "./dist/index.cjs.js"
     },
-    "./themes/microvideo": {
-      "browser": "./dist/themes/microvideo/index.mjs",
-      "import": "./dist/themes/microvideo/index.mjs",
-      "require": "./dist/themes/microvideo/index.cjs.js",
-      "default": "./dist/themes/microvideo/index.cjs.js"
-    },
-    "./themes/minimal": {
-      "browser": "./dist/themes/minimal/index.mjs",
-      "import": "./dist/themes/minimal/index.mjs",
-      "require": "./dist/themes/minimal/index.cjs.js",
-      "default": "./dist/themes/minimal/index.cjs.js"
-    },
-    "./themes/classic": {
-      "browser": "./dist/themes/classic/index.mjs",
-      "import": "./dist/themes/classic/index.mjs",
-      "require": "./dist/themes/classic/index.cjs.js",
-      "default": "./dist/themes/classic/index.cjs.js"
-    },
-    "./themes/gerwig": {
-      "browser": "./dist/themes/gerwig/index.mjs",
-      "import": "./dist/themes/gerwig/index.mjs",
-      "require": "./dist/themes/gerwig/index.cjs.js",
-      "default": "./dist/themes/gerwig/index.cjs.js"
+    "./themes/*": {
```

Comment:
question(blocking): Did you confirm this refactor doesn't have any issues with various build tools? if not, this feels out of scope.